### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by atomicbird** [wwdc-family/app#47]
I ran through the steps in the README and it seemed to be working until the last step, which fails with the following messages. Any suggestions?

    === BUILD TARGET GTMSessionFetcher OF PROJECT Pods WITH CONFIGURATION Debug ===

    Check dependencies
    The file “GTMSessionFetcher.xcconfig” couldn’t be opened because there is no such file. (/tmp/wwdc-family/ios/Pods/Target Support Files/GTMSessionFetcher/GTMSessionFetcher.xcconfig)
    error: unable to read module map contents from 'Target Support Files/GTMSessionFetcher/GTMSessionFetcher.modulemap': Error Domain=NSCocoaErrorDomain Code=260 "The file “GTMSessionFetcher.modulemap” couldn’t be opened because there is no such file." UserInfo={NSFilePath=/tmp/wwdc-family/ios/Pods/Target Support Files/GTMSessionFetcher/GTMSessionFetcher.modulemap, NSUnderlyingError=0x7f94d3405080 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}

    ** BUILD FAILED **

    The following build commands failed:
        Check dependencies
    (1 failure)

    Installing build/Build/Products/Debug-iphonesimulator/wwdcfamily.app
    An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=2):
    Failed to install the requested application
    An application bundle was not found at the provided path.
    Provide a valid path to the desired application bundle.
    Print: Entry, ":CFBundleIdentifier", Does Not Exist

    Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/wwdcfamily.app/Info.plist
    Print: Entry, ":CFBundleIdentifier", Does Not Exist

**Comment by KrauseFx** [comment]
I haven't seen that error, I don't know, maybe it has something to do with the version of react-native?

**Comment by ShylahChiera** [comment]
Hey @atomicbird ! Have you resolved your issue? I've come across the identical build failure, any word?

**Comment by atomicbird** [comment]
No. When app continued its high battery use, and when it proved impossible to build the app, I lost interest in the project.

**Comment by ShylahChiera** [comment]
I ended up running `pod init` then `pod install`, it recreated my podfile and then compiled just fine.

**Comment by KrauseFx** [comment]
Feel free to submit a PR to update the installation instructions 👍  Thanks
